src/marketing/price-rules-catalog-create.md

```diff
@@ -84,7 +84,7 @@ Most of the available conditions are based on existing attribute values. To appl
 
       The selected condition appears in the statement, followed by two more bold links. The options differ depending on the condition attribute you select. The statement now says:
 
-         If **ALL** of these conditions are **TRUE**: </ br>Attribute Set **is** …
+        If **ALL** of these conditions are **TRUE**: <br/>Attribute Set **is** …
 
    - Click **is** and choose the comparison operator that describes the condition to be met. These options may include an option for different comparisons, such as matching values, not including or including at least one of a value, and great than, equal to, and less than a numerical amount. In this example, the options are `is` and `is not`.
```

Comment:
"greater than"

(When I use the suggestion tool, GH changes the spacing at the beginning of the paragraph.)